crates/aptos/README.md

```diff
@@ -20,12 +20,16 @@ You will need the `cargo` package manager to install the `aptos` CLI tool.  Foll
 source $HOME/.cargo/env
 ```
 ### Install the `aptos` CLI
-1. Install the `aptos` CLI tool by running the below command.  You can run this command from any directory.  The `aptos`
+1. Install dependencies before compiling:
+   1. For Debian or Ubuntu: `sudo apt install build-essential pkg-config openssl libssl-dev libclang-dev`.
+   2. For RHEL or Centos: `sudo yum install pkgconfig openssl openssl-devel clang`.
+   3. For others: please manually install `pkg-config` `openssl`, `libssl` and `libclang`.
```

Comment:
You could probably also mention the `scripts/devsetup.sh` in the repo, but that might make it more complicated.  @rajkaramchedu is this clear enough?

Reply:
For step 3, can we add the command example to manually install? For example, we can say, "npm pkg-config" (if that is correct), then the user will understand it better.

Reply:
Obviously "for others" cover a whole gamut but "npm" will give the user a clear idea what we mean.

Reply:
Updated, thanks for you advice!